example/default/converter_test.go

```diff
@@ -0,0 +1,32 @@
+package example_test
+
+import (
+	"testing"
+
+	default1 "github.com/jmattheis/goverter/example/default"
+	"github.com/jmattheis/goverter/example/default/generated"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestConverterSuccess(t *testing.T) {
+	var c default1.Converter = &generated.ConverterImpl{}
+
+	input := default1.Input{
+		Age:  20,
+		Name: p("tester"),
+	}
+
+	output := c.Convert(&input)
+
+	expected := &default1.Output{
+		Age:  42,
```

Comment:
This is working as intended. With goverter:default the default values are defined. in this case age=42. The input struct in this case:
```
input := default1.Input{
	Age:  20,
	Name: p("tester"),
}
```
age=20 overrides the default values, because the default is only used when the value isn't set on the input struct.

Reply:
how about when I add ignore on Age then, should be `age=42` now right?

Reply:
It should be set by the constructor specified by the default tag right?

```
func NewOutput() *Output {
	name := "jmattheis"
	return &Output{
		Age:  42,
		Name: &name,
	}
}```